druid/src/core.rs

```diff
@@ -75,11 +76,15 @@ pub(crate) struct BaseState {
     /// drop shadows or overflowing text.
     pub(crate) paint_insets: Insets,
 
-    // TODO: consider using bitflags for the booleans.
+    // The region that needs to be repainted, relative to the origin of `layout_rect`.
```

Comment:
a distinction I make sometimes (borrowed from cocoa) is between 'frame' and 'bounds'; 'frame' is a rectangle in the parent's coordinate space, and 'bounds' is a rectangle in the current widget's coordinate space. In that sense, this is a region relative to the widget's bounds.